posthog/api/signup.py

```diff
@@ -83,43 +87,12 @@ def enable_new_onboarding(self, user: Optional[User] = None) -> bool:
         return posthoganalytics.feature_enabled("new-onboarding-2822", user.distinct_id) or settings.DEBUG
 
 
-class SocialSignupSerializer(serializers.Serializer):
```

Comment:
This class & `SocialSignupViewset` (below) are only moved verbatim to the bottom of the file for organization purposes.